bin/resty

```diff
@@ -233,7 +248,7 @@ $main_include_directives
 http {
     access_log off;
     lua_socket_log_errors off;
-    resolver @nameservers;
+    resolver @nameservers ipv6=off;
```

Comment:
this might be undesired but necessary for the `www.google.com` resolution test. Let me know if inappropriate.